internal/schema/stacks/1.9/orchestrate_block.go

```diff
@@ -6,13 +6,25 @@ package schema
 import (
 	"github.com/hashicorp/hcl-lang/lang"
 	"github.com/hashicorp/hcl-lang/schema"
+	"github.com/hashicorp/terraform-schema/internal/schema/refscope"
 	"github.com/hashicorp/terraform-schema/internal/schema/tokmod"
 	"github.com/zclconf/go-cty/cty"
 )
 
 func orchestrateBlockSchema() *schema.BlockSchema {
 	return &schema.BlockSchema{
 		Description: lang.PlainText("Defines an orchestration rule, such as a rule for when to auto-approve one or more deployments in the stack to be evaluated after a plan or apply operation. These rules allow you to define the behavior of various aspects of the stack in code, and make managing large numbers of deployments more manageable. The block labels include the rule type and the rule name, which together must be unique within the stack"),
+		Address: &schema.BlockAddrSchema{
+			Steps: []schema.AddrStep{
+				schema.StaticStep{Name: "orchestrate"},
+				schema.LabelStep{Index: 0},
+				schema.LabelStep{Index: 1},
```

Comment:
Oh, it's possible to reference these blocks? Or what do we need the address for? (just curious, I'm not up-to-date here 😄)